src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.header__nav-list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  text-transform: uppercase;
+}
+
+.header__nav-link {
+  position: relative;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  color: inherit;
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  @include animation-hover(color, $blue-accent);
```

Comment:
The `animation-hover` mixin is used here, but its definition is not provided in this file. Ensure that the mixin is defined in `mixins.scss` and that it follows the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) for mixins. If it is not defined or does not follow the code style rules, it should be updated accordingly.